MAVProxy/mavproxy.py

```diff
@@ -464,6 +464,11 @@ def import_package(name):
     """Given a package name like 'foo.bar.quux', imports the package
     and returns the desired module."""
     import zipimport
+
+    # Set the modules path to search for modules
+    src_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'modules')
```

Comment:
If MAVProxy is installed - aren't we essentially adding a random path to the search path here?